f/medical/fn_famMedswap.sqf

```diff
@@ -40,50 +40,43 @@ _itemsNeedUpdate = false;
 {
 	_itemReplaced = false;
 	if (_x == "FirstAidKit" || {[configFile >> "CfgWeapons" >> _x >> "ItemInfo","type",-1] call BIS_fnc_returnConfigEntry == 401}) then {
+		_itemType = _x;
 		_itemsQty = _existingCargo #1 #_forEachIndex;
 		_itemsCargo pushback ["Bandage",round _itemsQty * .75];
-		_itemsCargo pushback ["FirstAidKit",round _itemsQty * .25];
-		_itemReplaced = true;
-		_itemsNeedUpdate = true;
-	};
-	if (_x != "Medikit" && {[configFile >> "CfgWeapons" >> _x >> "ItemInfo","type",-1] call BIS_fnc_returnConfigEntry == 619}) then {
-		_itemsCargo pushback "Medikit";
+		_itemsCargo pushback [_itemType,round _itemsQty * .25];
 		_itemReplaced = true;
 		_itemsNeedUpdate = true;
 	};
 	if (!_itemReplaced) then {
 		_itemsCargo pushBack [_x,1];
 	};
 } forEach (_existingCargo #0);
+
 if (_itemsNeedUpdate) then {
 	clearItemCargoGlobal _obj;
 	{
-		_obj addItemCargoGlobal [_x #0,_x #1];
+		_obj addItemCargoGlobal _x;
 	} forEach _itemsCargo;
 };
 
-if (vehicle _obj == _obj) exitWith {}; 
 // ====================================================================================
-
+/*
```

Comment:
This section is commented out? Can it be deleted?

Reply:
@costno

Reply:
Yes